lib/ast/node.rb

```diff
@@ -177,9 +177,7 @@ def append(element)
     #     p value    # => (integer 1)
     #
     # @return [Array]
-    def to_a
-      children
-    end
+    alias to_a children
```

Comment:
What if you left the method in place, but afterwards overwritten it with `undef` and `alias`? Then both Ruby and Yard are happy.